### ISSUE TITLE
(Code Provided) Transfer learning using ResNet50 and SCS

### ISSUE THREAD

**Issue by sleepingcat4** [brohrer/sharpened-cosine-similarity#20]
Hi, Brohrer

I came across a this repository a few days ago. And saw no one tried to perform image classification using a combination of transfer learning and scs. That's why, I ran a small experiment and wrote a notebook showing how transfer learning and scs combination can result in **91% validation accuracy and 98% training accuracy.** https://github.com/sleepingcat4/resnet-scs

While doing it, I had wrote a Python library: https://pypi.org/project/sharpcosine/0.1.1/ that can be used to use the Keras scs layers. Will be grateful if you accept my Pull request and add them to the README.md file. 

Thanks! :)
